index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <link rel="stylesheet" href="styles.css">
+        <title>first mobile app</title>
+    </head>
+
+    <body>
+        <section class="main-page">
+            <header class="main-header">
+                <nav>
+                    <div class="header-text">John Doe</div>
+                </nav>
+                <nav class="union">
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                    <div class="vector"></div>
+                </nav>
```

Comment:
- [ ] This nav should have the hamburger in it. Something like the image below.
![image](https://user-images.githubusercontent.com/86200356/205280586-6885d9ab-5e08-48d3-81db-be873d850a8c.png)

Reply:
done

Reply:
dear @Olamarx should menu icon create with image? can I make it with nav tag?